.github/workflows/update_lite_galata_references.yaml

```
@@ -0,0 +1,111 @@
+name: Update Lite Galata References
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  update-snapshots:
+    if: >
+      (
+        github.event.comment.author_association == 'OWNER' ||
+        github.event.comment.author_association == 'COLLABORATOR' ||
+        github.event.comment.author_association == 'MEMBER'
+      ) && github.event.issue.pull_request && contains(github.event.comment.body, 'please update litesnaps')
```

Let's keep the same triggering rule

```suggestion
      ) && github.event.issue.pull_request && contains(github.event.comment.body, 'please update snapshots')
```

---

I wonder if it'd be a conflict